js/my_first_tour.js

```diff
@@ -0,0 +1,75 @@
+// Define the tour!
+var tour = {
+  id: "hello-hopscotch",
+  steps: [
+    {
+      title: "Navigation improvements!",
```

Comment:
I am not attached to the text in the titles, contents, buttons, etc. I am sure others can come up with better text.